docs/experimental.md

```diff
@@ -0,0 +1,11 @@
+# Experimental features
+
+Enable experimental features with:
+
+`export KOPS_FEATURE_FLAGS=`
+
+The following experimental features are currently available:
+
+* `+VSphereCloudProvider` - Enable vSphere cloud provider.
+* `+DrainAndValidateRollingUpdate` - Enable drain and validate for rolling updates.
+* `+EnableExternalDNS` - Enable external-dns with default settings (ingress sources only).
```

Comment:
Are we always going to want to deploy this in the future? We probably want a API level Boolean to deploy the feature. For instance, we may not want this feature when on vSphere.